src/extension.ts

```diff
@@ -80,9 +83,10 @@ class RunOnSaveExtension {
 	}
 
 	private get _execOption(): {shell: string} {
-		if (this.shell) {
-			return {shell: this.shell};
-		}
+		return {
+			shell: this.shell,
+			cwd: vscode.workspace.rootPath
```

Comment:
This causes a TypeScript compiler error due to a mismatch in the return type.

Reply:
Fixed